docs/knowledgebase/runtime/macros.md

```diff
@@ -63,27 +81,27 @@ decl_storage! {
 
 **What It Does**
 
-This macro takes a succinct statement of: 
+This macro takes a succinct statement of:
 
 ```
 #vis #name get(fn #getter) config(#field_name) build(#closure): #type = #default;
 ```
 
-Replacing it with a new struct type defined by `#name` and have it implement the data type storage trait. 
+Replacing it with a new struct type defined by `#name` and have it implement the data type storage
+trait.
 
-The macro also declare and implements `Store` trait to set up the pallet to have storage space, and implements getters on `Module` struct defined by `decl_module!`.
+The macro also declares and implements `Store` traits to set up the pallet to have storage space, and
+implements getters on the `Module` struct defined by `decl_module!`.
 
-**Doc. and Example**
+**Docs and Notes**
 
-  - [API Documentation](https://substrate.dev/rustdocs/v2.0.0-rc6/frame_support/macro.decl_storage.html)
-  - [Macro Definition](https://github.com/paritytech/substrate/blob/v2.0.0-rc5/frame/support/procedural/src/lib.rs#L236-L238)
-  - Macro Expansion Example: [original](https://gist.github.com/jimmychu0807/c4a88ec8e0342ee9f4e14bd26287324e#file-pallet-template-rs-L30-L40), [expanded](https://gist.github.com/jimmychu0807/c4a88ec8e0342ee9f4e14bd26287324e#file-pallet-template-expanded-rs-L19-L164)
-
-**Macro Implementation Notes**
-
-  - `Store` trait is declared with each storage name, thus becoming an associated type inside the trait.
-  - `Module` implementation contains the getter function and metadata of each storage item.
-  - Each of the storage items becomes a struct. Above in the expanded code `Something` struct type [is declared and implements `StorageValue<u32>` trait](https://gist.github.com/jimmychu0807/c4a88ec8e0342ee9f4e14bd26287324e#file-pallet-template-expanded-rs-L144-L164) and the data type specified in the macro.
+- [API Documentation](https://substrate.dev/rustdocs/v2.0.0-rc6/frame_support/macro.decl_storage.html)
+- `Store` trait is declared with each storage name, thus becoming an associated type inside the
+  trait.
+- `Module` implementation contains the getter function and metadata of each storage item.
+- Each of the storage items becomes a struct. Above in the expanded code `Something` struct type
+  [is declared and implements `StorageValue<u32>` trait](https://gist.github.com/jimmychu0807/c4a88ec8e0342ee9f4e14bd26287324e#file-pallet-template-expanded-rs-L144-L164)
```

Comment:
We need to find a better way than pointing to personal gists.

Reply:
Yes, I agree. Is the Gist here potentially out-of-date since it may be an expanded version of `rc5` or something? We would need a strategy around making sure this stays up-to-date.

Reply:
Yeah, for sure. I don't think that it contains updated code.